health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/ConfirmationPopUp.js

```
@@ -0,0 +1,87 @@
+    const handleSave = async () => {
+
+        setIsSubmitting(true);
+
+        // Call the API using the mutation hook
+        await mutation.mutate(
+            {
+                body: requestPayload
+            },
+            {
+                onSuccess: (data) => {
+                    onSuccess && onSuccess(data); // Call the onSuccess callback if provided
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)